src/Sylius/Bundle/ApiBundle/Resources/config/app/config.yaml

```
@@ -1,6 +1,9 @@
 # This file is part of the Sylius package.
 # (c) Paweł Jędrzejewski
 
+parameters:
+    sylius_api.authorization_header: AuthorizationTestHeader
```

```suggestion
    sylius_api.authorization_header: AuthorizationTestHeader
```
I added it temporary to check tests